### PR TITLE
Fix the code box issue in the `GetStarted` page of the `Playground` project on the small screen (#1987)

### LINKED ISSUE #1987: `GetStarted` page issue in the `Playground` project
The code text displayed in the box doesn't break in small screen sizes and will get out of the box:
freesnippingtool com_capture_20220509141441

### PR DESCRIPTION
**Pull request by sajadkardel**
this closes #1987

### CODE REVIEW

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/GetStartedPage.razor.scss [hunk 1]
```diff
@@ -188,6 +188,11 @@
     line-height: rem(24px);
     color: $B2Color;
     background-color: $B6Color;
+    overflow: auto;
+
+    &::-webkit-scrollbar:horizontal {
+        height: rem(6px);
+    }
```
Comment (msynk) [inline src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/GetStartedPage.razor.scss:195]: shouldn't we move these kinds of CSS properties into a more basic file?
@maryamhdr what do you think?

**Comment by msynk** [comment]
@maryamhdr plz review

File: src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/GetStartedPage.razor.scss (hunk 1 above)
Comment (maryamhdr) [reply src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/GetStartedPage.razor.scss:195]: This is not the correct solution...Let have a meeting @sajadkardel

**Review by msynk — CHANGES_REQUESTED** [review]

**Review by msynk — APPROVED** [review]

**Comment by msynk** [comment]
@maryamhdr review?

**Review by maryamhdr — APPROVED** [review]

**Pull request merged**